packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -1376,6 +1390,13 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
    * Handles dismissing (cancelling) the menu
    */
   private _onDismiss = (): void => {
+    // In persistMode we need to simulate callout layer mount
+    // since that only happens once. We do it on dismiss since
+    // it works either way.
+    if (this.props.persistMenu) {
+      this._onCalloutLayerMounted();
```

Comment:
Hmm I wonder if a name like `_resetMenuState` might be more clear?

Reply:
I am thinking since everything is ready to go I will check in, if you feel strongly I can submit a fix tomorrow.